assembl/scripts/gen_chatbot_css.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh
+: ${botBackgroundColor:='#120D9C'}
+: ${botTextColor:='white'}
+: ${botTimeColor:='lightgrey'}
+
+: ${humanBackgroundColor:='#b4f4be'}
+: ${humanTextColor:='black'}
+: ${humanTimeColor:='grey'}
+
+: ${copier:=$(
+    if command -v xclip 2>/dev/null; then :
+    elif command -v pbcopy 2>/dev/null; then :
+    elif command -v less 2>/dev/null; then :
+    else echo cat fi
```

Comment:
`:` is the bash built-in for true, right?
Why this complex conditions instead of the following:
`command -v xclip 2>/dev/null || command -v pbcopy 2>/dev/null || command -v less 2>/dev/null || echo cat`
It should be equivalent I think.

Reply:
it's the bash builtin for noop but it's the same as true I think

Reply:
I don't think I can replace with an or chain, it's not the same behaviour as javascript:
```
(venv)  ✘ Norman@bluenove-44  ~/assembl   motion-chatbot-own-frame  false || true || echo bli && false || echo bla
bla
```

Reply:
your version of the switch seems to work, will replace with it. I wrote it like this because sh && and || is a bit weird some times